README.md

```diff
@@ -303,7 +329,7 @@ signIn(callback?: (response: BasicUserInfo) => void, config?: SignInConfig, auth
 
 #### Description
 
-As the name implies, this method is used to sign-in users. This method will have to be called twice to implement the two phases of the authentication process. The first phase generates generates the authorization URl and takes the user to the single-sign-on page of the identity server, while second phase triggers the token request to complete the authentication process. So, this method should be called when initiating authentication and when the user is redirected back to the app after authentication themselves with the server.
+As the name implies, this method is used to sign-in users. This method will have to be called twice to implement the two phases of the authentication process. The first phase generates generates the authorization URl and takes the user to the single-sign-on page of the Asgardeo, while second phase triggers the token request to complete the authentication process. So, this method should be called when initiating authentication and when the user is redirected back to the app after authentication themselves with the server.
```

Comment:
Identity server is a generic term here. Should we change this?

Reply:
Fixed with 3ab5a77eb6f06027c2ab6e93fbd0c7368c71a57e